.packit.yml

```diff
@@ -0,0 +1,18 @@
+upstream_package_name: subscription-manager
+downstream_package_name: subscription-manager
+specfile_path: subscription-manager.spec
+upstream_tag_template: "subscription-manager-{version}-1"
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    targets:
+      - centos-stream-9
+
+  - job: copr_build
+    trigger: commit
+    branch: main
+    owner: "@yggdrasil"
+    project: latest
+    targets:
+      - centos-stream-9
```

Comment:
the `main` branch is already built in the `@yggdrasil/latest` copr, so this needs to be removed for now

Reply:
Oh, forgot to update parameter after cherry-picking. Shouldn't we change the branch to `subscription-manager-1.29`? The documentation https://packit.dev/docs/configuration/upstream/copr_build says: "branch - the name of the branch we want to build for when using commit trigger (defaults to the repository's default branch) or target branch when using pull_request trigger (default behaviour is reacting to all pull requests in the repository)."

Reply:
While the branch is not correct too, the problem I pointed out is about the copr in which build pushed commits. The `@yggdrasil/latest` copr is already used to build the `main` branch.

Reply:
I see your point... The `@yggdrasil/latest` copr is already little bit messy. We push builds there from `main` branch for RHEL9 or even from RHEL8 machines (build on RHEL9 and RHEL8 machines), which IMHO does not make much sense, because code from `main` branch will never be used on RHEL8 or RHEL9.

Can we create project `centos-stream-9` and `centos-stream-8` for builds for "release" branches?

Reply:
> Can we create projects `rhel-9` and `rhel-8` on copr and push there builds from "release" branches?

Let's do that as separate step; at least building PRs in packit will be enough as first step.